app/services/licences/fetch-licence-version-purpose-condition.service.js

```diff
@@ -10,45 +10,74 @@ const LicenceVersionPurposeCondtionModel = require('../../models/licence-version
 /**
  * Fetch the matching licence version purpose conditions and return data needed for the view licence page
  *
- * @param {string} id The UUID for the licence_version_purpose_id to fetch
+ * @param {object} ids The object contains the UUIDs for the licence_version_purpose_id and the purpose_id
```

Comment:
If it's an array of something you can do this with the JSDoc

```suggestion
 * @param {Object[]} ids - The object contains the UUIDs for the `licence_version_purpose_id` and the `purpose_id`
```